accounts_backend/views/mozilla_accounts_multi_service_dau.view.lkml

```diff
@@ -0,0 +1,80 @@
+view: mozilla_accounts_multi_service_dau {
+
+  derived_table: {
+
+    sql:
+    SELECT
+      DATE(e.submission_timestamp) as submission_date,
+      e.metrics.string.account_user_id_sha256 AS user_id,
```

Comment:
- Suggest keeping some indication in the column name and dimension name that this is a hashed value.
- Recommend adding a `WHERE` clause filter to only include records with non-null user ID hash values.